aspnetcore/host-and-deploy/aspnet-core-module.md

```diff
@@ -25,11 +25,15 @@ There are trade-offs between each of the hosting models. By default, the in-proc
 
 ## Install ASP.NET Core Module
 
+The ASP.NET Core Module is installed with the .NET Core Runtime from the [.NET Core Hosting Bundle](xref:host-and-deploy/iis/hosting-bundle). The ASP.NET Core Module is forward and backward compatible with LTS releases of .NET.
+
+Breaking changes or security advisories that apply to the ASP.NET Core Module or .NET Core Hosting Bundle are reported on the [Announcements repo](https://github.com/aspnet/Announcements/issues). Announcements can be filtered with `label:{VERSION}`, where the `{VERSION}` placeholder is a .NET release version (for example, `label:6.0.0`).
```

Comment:
Only a suggestion. Filtered made me look in the **Filters**. Mine is probably too verbose, folks can figure it out.

```suggestion
Breaking changes or security advisories that apply to the ASP.NET Core Module or .NET Core Hosting Bundle are reported on the [Announcements repo](https://github.com/aspnet/Announcements/issues). Announcements displayed can be limited to a specific version the with `label:{VERSION}`, where the `{VERSION}` placeholder is a .NET release version. For example, selecting **6.0.0** from the **Label** fiters displays only issues for the 6.0.0 release.
```

Reply:
Yes on the use of the DDL. That's a better way to go.

Reply:
S&M, include is over-kill